packages/settings-plugins/settings-plugins.spec

```diff
@@ -139,6 +151,11 @@ Conflicts: %{_cross_os}settings-plugin(any)
   -p settings-plugin-vmware-k8s \
   %{nil}
 
+%cargo_build --manifest-path %{_builddir}/sources/Cargo.toml \
+  -p settings-plugin-aws-k8s-nvidia \
+  %{nil}
```

Comment:
Would be good to add a comment as to why this is built separately (because it needs a different feature flag.)

Can you demonstrate that `settings-plugin-aws-k8s` is built without the `nvidia-device-plugin` feature?

Reply:
I did not understand what do you mean by "Can you demonstrate that settings-plugin-aws-k8s is built without the nvidia-device-plugin feature?"
The feature flag is provided in the cargo file of that package.

Reply:
You have two settings plugins:
* `settings-plugin-aws-k8s` - should be built without the feature flag
* `settings-plugin-aws-k8s-nvidia` - should be built with the feature flag

The risk is that with [feature unification](https://doc.rust-lang.org/cargo/reference/features.html#feature-unification) you end up with this behavior:
> When a dependency is used by multiple packages, Cargo will use the union of all features enabled on that dependency when building it

So it's not obvious that `settings-plugin-aws-k8s` would be built without the feature flag and you should take steps to confirm that is actually the case.

Reply:
I have changed the build script to build settings-plugin-aws-k8sand settings-plugin-aws-k8s-nvidia separately to avoid feature flag unification. Also added a comment for that.